test/quantization/pt2e/test_x86inductor_quantizer.py

```diff
@@ -400,3 +417,80 @@ def test_conv2d_serials_binary_unary_with_quantizer_api(self):
                 node_occurrence,
                 node_list,
             )
+
+    @skipIfNoX86
+    def test_linear_with_quantizer_api(self):
```

Comment:
nit: you can remove `_with_quantizer_api` now

Reply:
Thanks. It's removed.